home/templates/admin_create.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Can you use the base.html, and inherit its content and style here? You can check how it's being used on the other templates.